Gruntfile.js

```diff
@@ -302,13 +312,17 @@ module.exports = function(grunt) {
         files: ['static/css/*.less', '<%= loc.src %>/static/js/**/*.js', 'Gruntfile.js'],
         tasks: ['default']
       },
+      cssjs: {
+        files: ['static/css/*.less', '<%= loc.src %>/static/js/**/*.js'],
+        tasks: ['css', 'js']
+      },
       css: {
         files: ['static/css/*.less'],
         tasks: ['css']
       },
-      cssjs: {
-        files: ['static/css/*.less', '<%= loc.src %>/static/js/**/*.js'],
-        tasks: ['css', 'js']
+      js: {
```

Comment:
Thank you! I've been using this locally but waited to port it back till you had finished.